lighthouse-core/gather/gatherers/seo/tap-targets.js

```diff
@@ -257,10 +299,20 @@ function gatherTapTargets() {
   /** @type {LH.Artifacts.TapTarget[]} */
   const targets = [];
 
+  // Capture element positions relative to the top of the page
+  window.scrollTo(0, 0);
+
   /** @type {Element[]} */
   // @ts-ignore - getElementsInDocument put into scope via stringification
   const tapTargetElements = getElementsInDocument(tapTargetsSelector);
 
+
+  /** @type {{
+    tapTargetElement: Element,
+    largestRectCenterPoint: {x: number, y: number},
+    visibleClientRects: ClientRect[]
+  }[]} */
+  const enhancedTapTargets = [];
```

Comment:
i don't love `enhanced` since I'm not sure what that means but I don't have a ton of other suggestions


`tapTargetsWithClientRects` or something maybe?